tools/gstcase.c

```diff
@@ -283,11 +283,8 @@ gst_case_get_pipeline_string (GstCase * cas)
 {
   gboolean is_audiostream = cas->serve_type == GST_SERVE_AUDIO_STREAM;
   GString *desc = g_string_new ("");
-  GString *caps = g_string_new ("");
-
-  if (!is_audiostream)
-    g_string_append_printf (caps, "video/x-raw,width=%d,height=%d", cas->width,
-        cas->height);
+  const gchar *caps =
+      is_audiostream ? "" : gst_switch_server_get_video_caps_str ();
```

Comment:
The audio caps are;
`audio/x-raw,rate=48000,channels=2,format=S16LE,layout=interleaved`